nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -143,11 +148,11 @@ def __init__(self, model, config_list, optimizer=None):
                     types of nn.module you want to apply quantization, eg. 'Conv2d'
         """
         super().__init__(model, config_list, optimizer)
-        self.steps = 1
         modules_to_compress = self.get_modules_to_compress()
+        self.bound_model.register_buffer("steps", torch.Tensor([1]))
         for layer, config in modules_to_compress:
-            layer.module.register_buffer("zero_point", None)
-            layer.module.register_buffer("scale", None)
+            layer.module.register_buffer("zero_point", torch.Tensor([0.0]))
```

Comment:
If we just use 0-dimension definition here, would it be more simple and we don't need to add conversion?

Reply:
define 0-dimension does save some work,defining it to 1-dimension tensor is just for keeping consitent with other buffered variables